test/pgtap.sql

```diff
@@ -98,6 +98,7 @@ SELECT is_partitioned('pgstac'::name,'items'::name);
 SELECT has_function('pgstac'::name, 'get_item', ARRAY['text']);
 SELECT has_function('pgstac'::name, 'delete_item', ARRAY['text']);
 SELECT has_function('pgstac'::name, 'create_item', ARRAY['jsonb']);
+SELECT has_function('pgstac'::name, 'create_items', ARRAY['jsonb']);
 SELECT has_function('pgstac'::name, 'update_item', ARRAY['jsonb']);
 SELECT has_function('pgstac'::name, 'upsert_item', ARRAY['jsonb']);
```

Comment:
Maybe we should also have an `upsert_items`? 🤷‍♂️

Reply:
@kylebarron there is also the third option which in the pypgstac loader I have as "ignore_duplicates" where it will not overwrite any data, but it will not fail if the id already exists.